docs/ai-applications/athenaserving.md

```diff
@@ -0,0 +1,134 @@
+# AthenaServing Framework
+
+## 框架介绍
+
+`AthenaServing Framework(下简称ASF)` AI推理服务框架依托科大讯飞多年的AI算法引擎云服务化经验及云原生的不断探索实践,不仅可以满足引擎云服务化后,服务的稳定性,也可以通过`ASF`享受到相关云原生组件的方便与快捷。AI算法引擎开发者可以专注于算法的演进与研究,无需分心进行硬件资源的管理及云服务化的诸多开发运维工作。
+
+`ASF`是一个专为AI能力开发者打造的AI算法引擎的无服务全托管式平台框架，您可以通过集成 `ASF` 中提供的插件，快速的部署AI算法引擎，并使用网络、分发策略、数据处理等配套辅助系统。引擎托管平台致力于加速AI算法引擎云服务化，并借助云原生架构，为云服务的稳定提供多重保障，您无需关注底层基础设施及服务化相关的开发、治理和运维，即可高效、安全对引擎进行部署、升级、扩缩、运营和监控。
+
+目前部署 `ASF` 需要开发者掌握一定的K8s、helm 等相关知识，且安装部署依赖在线镜像仓库和 helm repo，在离线环境、各式各样的操作系统部署的需求面前有些乏力，sealos 以其`集群镜像`、`images-shim`等方案让应用可以让离线部署变得非常丝滑，无需任何额外操作，sealos 支持了 `ASF` 集群镜像后，让任何人在任何场景、任何环境无障碍交付 `ASF` , 仅需一条命令即可拉起一个 `ASF` 环境。用户可以在 `ASF`框架上集中部署自己的AI能力，对外提供HTTP API。
+
+
+## 框架架构
+
+
+![img](https://github.com/iflytek/proposals/blob/main/athenaloader/athena.png?raw=true)
+
+
+## 框架安装
+
+### 前置条件
+
+准备一台测试机(4c8G),硬盘>=20G即可
+
+
+### 安装
+
+1. 安装sealos.4.0
+
+``` wget -c https://sealyun-home.oss-cn-beijing.aliyuncs.com/sealos-4.0/latest/sealos-amd64 -O sealos &&  chmod +x sealos && mv sealos /usr/bin```
```

Comment:
There's a [docs contributing guide](https://github.com/labring/sealos/blob/main/CONTRIBUTING.md#docs-contribution) that requires code block to be like:

<pre>
```shell
wget ...
```
</pre>

or:

<pre>
```shell
$ wget ...
```
</pre>